client/components/ResponsiveFlex/ResponsiveFlex.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropsWithChildren, useEffect } from 'react';
+
+interface Props {
+  breakpoint: number;
+}
+
+function ResponsiveFlex({ breakpoint, children }: PropsWithChildren<Props>) {
```

Comment:
Please document this component. It's quite complex so let's write a little bit about why we want it and how it works.

Also, let's add it to Storybook.